antiplag/constants.py

```diff
@@ -1,4 +1,5 @@
 CONTENT_TYPE_FILE = "multipart/form-data"
-CONTENT_TYPE_TEXT = "text/plain"
+CONTENT_TYPE_TEXT = "application/json"
 
 TEXT_SUBMISSION_NAME = "Text submission"
+EMAIL_SENDER = "Antiplag <antiplag@antiplag.sk>"
```

Comment:
Does it work OK or do we need to adjust DNS entries?

Reply:
All works great, emails are received via the SendGrid APIs, never landed in spam, twice into promotions but then always into main email.

Reply:
Just impersonation from the email header I think.

Reply:
Working not, I don't think we have an inbox setup, we can add that to the next sprint as a task.